src/test/java/org/elasticsearch/index/mapper/timestamp/TimestampMappingTests.java

```diff
@@ -524,6 +525,87 @@ public void testMergingConflictsForIndexValues() throws Exception {
         }
     }
 
+    /**
+     * Test case for #9204
+     */
+    @Test
+    public void testMergingNullValues_9204() throws Exception {
```

Comment:
I don't think the test name needs the issue number.  The rest of the name explains what the test is checking.